package.json

```diff
@@ -49,7 +49,11 @@
       "hardenedRuntime": true,
       "entitlements": "assets/entitlements.mac.plist",
       "entitlementsInherit": "assets/entitlements.mac.plist",
-      "gatekeeperAssess": false
+      "gatekeeperAssess": false,
+      "extendInfo": {
+        "NSMicrophoneUsageDescription": "We need access to your microphone so people you talk to in a SugarTalk can hear you.",
```

Comment:
SugarTalk would like to access your microphone so that people can communicate by audio